cfgov/cfgov/settings/base.py

```diff
@@ -631,6 +627,12 @@
     "*.qualtrics.com",
 )
 
+# These specify valid media sources (e.g., MP3 files)
+CSP_IMG_SRC = (
```

Comment:
Isn't this duplicative of the `CSP_IMG_SRC` above? From the comment it seems like this should be `CSP_MEDIA_SRC`

Reply:
🤦 And this is why we have code review even for simple changes! :)

Reply:
Fixed. Thanks for the catch!